src/pages/cli/geo/geofencing.mdx

```diff
@@ -3,7 +3,9 @@ export const meta = {
   description: `Use Amplify CLI to create and manage collections of Geofences`,
 };
 
-Amplify's `geo` category enables you to create and manage collections of Geofences used to setup virtual geographic perimeters.
+A Geofence is a virtual perimeter for a real-world geographic area. Geofences contain points and vertices that form a closed boundary, which defines an area of interest. 
+Geofence collections store and manage one or multiple Geofences.
```

Comment:
How does Geofence collection manage geofences?

Reply:
This definition is taken from Location service docs for reference https://docs.aws.amazon.com/location/latest/developerguide/add-geofences.html. But let's remove `manage` to avoid confusion.